mutmut/cache.py

```diff
@@ -21,9 +21,11 @@
     from itertools import izip_longest as zip_longest
 
 if sys.version_info < (3, 0):   # pragma: no cover (python 2 specific)
+    from itertools import izip_longest as zip_longest
```

Comment:
Already fixed, see lines 18-21.

Reply:
We still have to do a python version check due to the usage of `text_type` should we move that import there (it explicitly hints to the usage of this import for backwards compatibility). Maybe not do this in this pr but maybe another.

Reply:
Maybe for now just remove these lines so we don't duplicate the fix, but I see your point.

In my experience compat imports are done with:
```
try:
    ... 
except ImportError:
    ...
``` 
while
```
if sys.version_info > ...:
    IOWrapper = _BaseIOWrapper
```
for example is used to set attributes to the module.

https://github.com/python/cpython/blob/2e438cc2554495b28480a3ffe5cdf41b6ab823a0/Tools/gdb/libpython.py#L53-L56

https://github.com/python/cpython/blob/9fb051f032c36b9f6086b79086b4d6b7755a3d70/Tools/ssl/multissltests.py#L31-L34


 Maybe someone else can add some input on this one.

Reply:
Might as well not change it (thus retaining master's solution). I prefer less branching, but, catching `ImportError` is a powerful catch all for this issue.